project/Dependencies.scala

```diff
@@ -11,7 +11,7 @@ object Dependencies {
 
   val workbenchUtilV    = "0.2-d97f551"
   val workbenchModelV   = "0.10-6800f3a"
-  val workbenchGoogleV  = "0.15-2fc79a3"
+  val workbenchGoogleV  = "0.16-16e3aab"
```

Comment:
This is really to get https://github.com/broadinstitute/workbench-libs/pull/90. I observed this `GoogleStorageDAO` error when 2 monitor actors were racing to delete the same cluster.